packages/tools/visual-tests/tests/sample-app.routes.js

```diff
@@ -314,11 +314,6 @@ ROUTES.set('table/badge-size', {
 		skipFailures: false,
 	},
 });
-ROUTES.set('table/render-cell', {
```

Comment:
Dieser Test hat angefangen Probleme in verschiedenen Themes zu machen, vermutlich aufgrund der Größe des Snapshots. Ich habe ihn entfernt, weil ich ihn für das visuelle Testing auch nicht unbedingt für notwendig erachte.